torch/distributed/distributed_c10d.py

```diff
@@ -3767,3 +3802,50 @@ def new_subgroups_by_enumeration(
                 logger.info("Rank {} is assigned to subgroup {}".format(rank, ranks))
 
     return cur_subgroup, subgroups
+
+
+def _try_find_pg_by_ranks_and_tag(tag: str, ranks: List[int]) -> ProcessGroup:
```

Comment:
nit: I would skip the "_try" in the function name.

Reply:
👍